### DISCUSSION TITLE
Binding factory default type

### DISCUSSION THREAD

**Discussion by assafcohen** [taocpp/json#69] [Ideas]
While using factory binding like:
        template<>
        struct traits<std::shared_ptr<action_msg> > : binding::factory<
                TAO_JSON_FACTORY_BIND1(smb_list_dir_msg),
                TAO_JSON_FACTORY_BIND1(get)
        > {        };
is it possible to get a default object when nothing fits the type list?
Thanks

**Comment by ColinH** [comment]
This is currently not possible, as we have not really considered this use case yet. It would require some changes to the current implementation.
